collectors/ebpf.plugin/ebpf_sync.c

```diff
@@ -256,35 +287,24 @@ static void ebpf_sync_cleanup(void *ptr)
 static void ebpf_create_sync_charts()
 {
     if (local_syscalls[NETDATA_SYNC_FSYNC_IDX].enabled || local_syscalls[NETDATA_SYNC_FDATASYNC_IDX].enabled)
-        ebpf_create_chart(NETDATA_EBPF_MEMORY_GROUP, NETDATA_EBPF_FILE_SYNC_CHART,
-                          "Monitor calls for <code>fsync(2)</code> and <code>fdatasync(2)</code>.",
-                          EBPF_COMMON_DIMENSION_CALL, NETDATA_EBPF_SYNC_SUBMENU, NULL,
-                          NETDATA_EBPF_CHART_TYPE_LINE, 21300,
-                          ebpf_create_global_dimension, &sync_counter_publish_aggregated[NETDATA_SYNC_FSYNC_IDX],
-                          2);
+        ebpf_create_sync_chart(NETDATA_EBPF_FILE_SYNC_CHART,
+                               "Monitor calls for <code>fsync(2)</code> and <code>fdatasync(2)</code>.", 21300,
```

Comment:
`<code>`

It doesn't work in the title, does it?

Reply:
At least on firefox it is working fine.